test/core/rules/config_test.py

```diff
@@ -21,8 +24,14 @@ def _eval(self, context):
         pass
 
 
+@document_fix_compatible
```

Comment:
This rule is intended to cause an endless loop of "fixes". Had to add this decorator so the new, smarter lint loop would actually call it every loop. (Now on pass 2 and beyond, it skips calling rules that don't declare they return fixes.)